packages/eslint-plugin-office-addins/src/utils/load.ts

```diff
@@ -41,21 +41,28 @@ function composeObjectExpressionPropertyIntoString(
   return composedProperty;
 }
 
-export function getLoadArgument(node: TSESTree.MemberExpression): string {
+export function getLoadArgument(
+  node: TSESTree.MemberExpression
+): string | undefined {
   node = findTopLevelExpression(node);
 
   if (
     isLoadFunction(node) &&
     node.parent?.type === TSESTree.AST_NODE_TYPES.CallExpression
   ) {
-    if (node.parent.arguments[0].type === TSESTree.AST_NODE_TYPES.Literal) {
-      return node.parent.arguments[0].value as string;
-    } else if (
-      node.parent.arguments[0].type === TSESTree.AST_NODE_TYPES.ObjectExpression
-    ) {
-      return composeObjectExpressionPropertyIntoString(
-        node.parent.arguments[0]
-      );
+    if (node.parent.arguments.length === 0) {
+      return undefined;
+    } else {
+      if (node.parent.arguments[0].type === TSESTree.AST_NODE_TYPES.Literal) {
+        return node.parent.arguments[0].value as string;
+      } else if (
+        node.parent.arguments[0].type ===
+        TSESTree.AST_NODE_TYPES.ObjectExpression
+      ) {
+        return composeObjectExpressionPropertyIntoString(
+          node.parent.arguments[0]
```

Comment:
I think I mentioned this before, but getting only the first property loose all checks of the other properties.  Do we have a separate bug for this?  Is there a plan to fix it?

Reply:
This is to see only the first argument of the load, which should be a string. So any string inside load will get caught by this `[0]`, even if the string represents multiples properties.

Example: `load("address, font");` still is only 1 string and all of it will be inside `arguments[0]`.

Reply:
I think I was getting this confused with the method that takes the string and pulls out the different properties . . . I guess this is the reverse of that where we take an object and make a string (note your example of load("address, font") would get caught by the previous if statement since it's already a literal string).

The thought and question still applies though really in a different location.  We are using this method to get the argument and use it places like call-sync-after-load to identify properties, but we are treating like a single property.  It's not going to be addressed here, but do we have a follow up for that.

Reply:
Sure makes sense. Yeah, I will later address the bug of us not supporting commas `load("x, y")` and I will look into previous rules to make sure they work for all types of loads.

Reply:
Is there a task or bug tracking this work that needs to be done?

Reply:
I was planning on doing it on this task:
load with multiple properties in the argument needs to be able to parse each property

I will add your comments in the description for us to not forget about supporting these other loads for all rules.